# Fix "too many arguments" error when $CC contains multiple words, e.g. when using ccache or distcc
--- configure.orig	2011-10-21 23:14:14.000000000 -0500
+++ configure	2011-11-15 18:10:21.000000000 -0600
@@ -14612,7 +14612,7 @@
 
 
 # Need to tell automake if Visual C++ is being used:
- if test x$CC = xcl.exe; then
+ if test "x$CC" = xcl.exe; then
   USING_CL_TRUE=
   USING_CL_FALSE='#'
 else
